### ISSUE TITLE
Module fails to load in React Native

### ISSUE THREAD

**Issue by mattslight** [knicklabs/lorem-ipsum.js#25]
In React Native 0.56, including lorem-ipsum fails.

This is due to lib/generator.js `requiring('os')` in the conditional block:-

```js
    if (!isReactNative && isNode) {
      suffix = require('os').EOL
    } else {
      suffix = '\n'
    }
```

`!isReactNative && isNode` correctly evaluates to false but React Native is still checking the haste cache for require('os').

<img width="328" alt="screenshot 2018-08-18 09 02 09" src="https://user-images.githubusercontent.com/1073639/44297204-69acf600-a2c5-11e8-8e89-d2fb17c36ad4.png">

**Comment by dubzzz** [comment]
I have been reported a similar issue with rollup build (my repo rely on lorem-ipsum.js internally). It seems the `require('os')` breaks the rollup build too.

See https://github.com/dubzzz/fast-check/issues/168

**Comment by knicklabs** [comment]
@mattslight I released version 1.0.6 which includes @dubzzz's fix.

**Comment by mattslight** [comment]
Brilliant - haven't tested yet as on vacation. Looked at @dubzzz commit - looks straight forward enough.

Thanks for the update.
